internal/pkg/api/cloudstack.go

```diff
@@ -67,6 +68,33 @@ func WithCloudStackManagementServer(value string) CloudStackFiller {
 	}
 }
 
+func WithCloudStackAz(az anywherev1.CloudStackAvailabilityZone) CloudStackFiller {
+	return func(config CloudStackConfig) {
+		config.datacenterConfig.Spec.AvailabilityZones = append(config.datacenterConfig.Spec.AvailabilityZones, az)
+	}
+}
+
+func WithFirstCloudStackAz(az anywherev1.CloudStackAvailabilityZone) CloudStackFiller {
+	return func(config CloudStackConfig) {
+		config.datacenterConfig.Spec.AvailabilityZones[0] = az
+	}
+}
+
+func WithoutCloudStackAz(az anywherev1.CloudStackAvailabilityZone) CloudStackFiller {
```

Comment:
Should this just pass in the name then?

Reply:
I am just wondering if this is needed vs having something like `RemoveCloudstackAzs` and allowing you to recreate them

Reply:
Refactored so that instead of removing one at a time, we instead remove all the AZ's, and then can add it back incrementally